merkle/map_verifier.go

```diff
@@ -32,13 +32,13 @@ func VerifyMapInclusionProof(index, leafHash, expectedRoot []byte, proof [][]byt
 	hBits := h.Size() * 8
 
 	if got, want := len(proof), hBits; got != want {
-		return fmt.Errorf("invalid proof length %d, expected %d", got, want)
+		return fmt.Errorf("invalid proof length %d, want %d", got, want)
```

Comment:
Can you update all the Errorf message strings to "got:, want:" format + several below.